### DISCUSSION TITLE
Sync column widths of 2 similar tables

### DISCUSSION THREAD

**Discussion by FuriousBranko** [saadeghi/daisyui#1821] [General]
Hello, currently I'm displaying 2 tables. one below the other.
They're using the same columns and data types.

But the 2nd data set for the 2nd table has a few names that are long thus it shifts the columns more to the right. 
( the first column contains Names, and there are 3 people that have longer names than the longest one in the 1st data set)

Can all columns be the same width in both tables but still be responsive? 

I'm using NextJS 13.

**Comment by OEvgeny** [comment]
You could manually set the width of the column which doesn't fit for both tables. Or you probably could merge two tables into a single one with multiple thead and tbody tags: https://play.tailwindcss.com/MMSFuQ71P5

**Reply by FuriousBranko** [reply]
Thank you for your input. though im not using 1 table component. each table is separated. meaning in the solution you provided if you close the 1st table and add the same tag to the 2 table you would see how it looks like for me [https://play.tailwindcss.com/5Px6x0eFaa](https://play.tailwindcss.com/5Px6x0eFaa)

looking at it I presume I need to use JS to get the widths and then re-apply them to the 2nd table.
